onchain/src/tests/username_store.cairo

```diff
@@ -0,0 +1,72 @@
+use username_store::UsernameStore;
+use username_store::{IUsernameStoreDispatcher, IUsernameStoreDispatcherTrait};
+use username::UserNameClaimErrors;
+
+
+
+#[cfg(test)]
+mod tests {
+
```

Comment:
Tests are not building or passing, make sure to use `scarb test` and resolve the issues.

Reply:
am working on a fixed.

Reply:
Alright